src/suites/cts/gpu_shader_test.ts

```diff
@@ -0,0 +1,31 @@
+import { GPUTest } from './gpu_test.js';
+
+// tslint:disable-next-line: no-any
+let glslangInstance: any = undefined;
+
+export class GPUShaderTest extends GPUTest {
+  async init(): Promise<void> {
+    await super.init();
+    if (!glslangInstance) {
+      const glslangPath = '../../glslang.js';
+      const glslangModule = (await import(glslangPath)).default;
+      await new Promise(resolve => {
+        // tslint:disable-next-line: no-any
+        glslangModule().then((module: any) => {
+          glslangInstance = module;
+          resolve();
+        });
+      });
+    }
+  }
+
+  makeShaderModule(type: 'f' | 'v' | 'c', source: string): GPUShaderModule {
+    return this.device.createShaderModule({ code: this.compile(type, source) });
+  }
+
+  private compile(type: 'f' | 'v' | 'c', source: string): Uint32Array {
```

Comment:
nit: Give `'f' | 'v' | 'c'` a name ?

Reply:
Done, figured out how to import the types from glslang-web and fixed all the anys too.